### DISCUSSION TITLE
Support for free llms?

### DISCUSSION THREAD

**Discussion by eyyyyyyy3** [Zhengqbbb/cz-git#203] [Ideas]
How about instead of just supporting OpenAI, also supporting free LLMs like llama(via ollama)?

**Comment by Zhengqbbb** [comment]
But the effect is not good, can have a try ~

![CleanShot 2024-11-19 at 20 50 17@2x](https://github.com/user-attachments/assets/a7488fd4-56e1-4c3a-ba8d-be56a0bee1ea)

**Reply by eyyyyyyy3** [reply]
Oh there is already support for that. Isn't the newest model suppa strong and stuff.

**Reply by Zhengqbbb** [reply]
Emmm, I haven't had time to test it yet, but the github model is already sufficient for me.

**Reply by eyyyyyyy3** [reply]
llama3.2 works nice  :+1:
